proto/services/property_svc/v1/property_svc.proto

```diff
@@ -2,52 +2,85 @@ syntax = "proto3";
 
 package proto.services.property_svc.v1;
 
+import "proto/services/property_svc/v1/types.proto";
+
 option go_package = "gen/proto/services/property-svc";
 
 // TODO: This proto is just an hello-world example and far from where it should be!
 
-enum FieldType {
-  FIELD_TYPE_UNSPECIFIED = 0;
-  FIELD_TYPE_TEXT = 1;
-  FIELD_TYPE_NUMBER = 2;
-  FIELD_TYPE_CHECKBOX = 3;
-  FIELD_TYPE_DATE = 4;
-  FIELD_TYPE_DATETIME = 5;
-  FIELD_TYPE_SELECT = 6;
-}
-
 //
 // CreateProperty
 //
-message CreatePropertyRequest{
-  string subject_id = 1;
-  string subject_type = 2;
-  FieldType field_type = 3;
-  string name = 4;
+message CreatePropertyRequest {
+  ViewContext context = 1;
+  SubjectType subject_type = 2; // @gotags: validate:"required"
+  FieldType field_type = 3; // @gotags: validate:"required"
+
+  string name = 4; // @gotags: validate:"required"
+  optional string description = 5; // TODO: validation needed?
+
+  // ID of set this Property should belong to
+  optional string set_id = 6;
+
+  optional bool always_include_for_current_context = 7;
+
+  oneof field_type_data {
+    bool none = 8;
+    SelectData select_data = 9;
+  }
+
+  message SelectData {
+    optional bool allow_freetext = 1;
+    repeated SelectOption options = 2;
+    message SelectOption {
+      string name = 1; // @gotags: validate:"required"
+      optional string description = 2; // TODO: validation needed?
+    }
+  }
 }
 
 message CreatePropertyResponse {
-  string id = 1;
+  string property_id = 1;
 }
 
 //
 // GetProperty
 //
 
 message GetPropertyRequest {
-  string id = 1;
+  string id = 1; // @gotags: validate:"uuid4"
+  ViewContext context = 2; // TODO: Why is that here?
 }
 
 message GetPropertyResponse {
   string id = 1;
-  string name = 2;
-  string subject_type = 3;
-  string subject_id = 4;
-  FieldType field_type = 5;
-
-  optional string description = 6;
-  optional bool is_archived = 7;
-  optional bool is_soft_required = 8;
+  SubjectType subject_type = 2;
+  FieldType field_type = 3;
+
+  string name = 4;
+  optional string description = 5;
+
+  bool is_archived = 6; //TODO: Is that still part of property model?
```

Comment:
This value can only be updated, it makes no sense to create a property directly as archived.